md/release-notes.md

```diff
@@ -7,3 +14,15 @@
  
  Only in the case you have created custom widgets using this silent library, you need to update them by adding UI Bootstrap js as an asset.
 
+
+### Full Java 8 support
+Bonita BPM 7.5.0+ is now fully Java 8 compliant.  
+Java 8 syntax and specific APIs can now be fully used in Connectors, Actor filters, Groovy Scripts, Custom REST APIs, Custom Event Handlers ...  
+As a consequence, JRE or JDK 8+ is required to run. JRE / JDK 7 is no longer supported to run Bonita BPM 7.5.0+
```

Comment:
Is it the purpose of the release-notes page to include a Troubleshooting section?